tests/e2e/upgrade_test.go

```diff
@@ -535,7 +535,9 @@ var _ = Describe("Upgrade", Label(tests.LabelUpgrade, tests.LabelNoOpenshift), O
 			podList, err := env.GetClusterPodList(upgradeNamespace, clusterName1)
 			Expect(err).ToNot(HaveOccurred())
 			for _, pod := range podList.Items {
-				Expect(pod.Status.ContainerStatuses[0].RestartCount).To(BeEquivalentTo(0))
+				if len(pod.Status.ContainerStatuses) > 0 {
```

Comment:
I think that at this point, if `len(pod.Status.ContainerStatuses) == 0`, we should raise an error.